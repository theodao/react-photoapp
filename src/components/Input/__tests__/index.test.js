import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { shallow } from 'enzyme';
import InputComponent from '../';

describe('Testing input component', () => {
  afterEach(cleanup);

  it('Should match snapshot', () => {
    const { asFragment } = render(<InputComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
