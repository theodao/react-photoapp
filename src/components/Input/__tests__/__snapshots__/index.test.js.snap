// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing input component Should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <input
      class="FormInput u-block u-widthFull u-webkitScrollbar"
      required=""
      style="font-size: 0.9rem;"
    />
  </div>
</DocumentFragment>
`;
